.eslintrc.cjs

```diff
@@ -8,7 +8,7 @@ const rules = {
   'node/prefer-global/process': ['error', 'always'],
   radix: 'off',
   'unicorn/no-array-reduce': 'off',
-  'no-console': ['error', { allow: ['warn', 'error'] }],
+  'no-console': 'off',
```

Comment:
should we change this to `warn`

Reply:
warn wasn't an option, but noticing in my code that it underlines it anyway (without erroring)

Reply:
oh ok sweet!